builder/amazon/common/access_config.go

```diff
@@ -65,8 +66,10 @@ func (c *AccessConfig) Config() (*aws.Config, error) {
 // the region from the instance metadata if possible.
 func (c *AccessConfig) Region() (string, error) {
 	if c.RawRegion != "" {
-		if valid := ValidateRegion(c.RawRegion); valid == false {
-			return "", fmt.Errorf("Not a valid region: %s", c.RawRegion)
+		if c.SkipValidation == false {
+			if valid := ValidateRegion(c.RawRegion); valid == false {
```

Comment:
I know this is a bit out of scope for your PR but I think if we rename this to `RegionIsValid` we can skip the variable and this whole section reads better.

E.g. `if RegionIsValid(c.RawRegion) ...`. Not sure if there are other implications to renaming this so feel free to ignore this point! I can leave it as a todo for myself after merging.